docs/manage/partitions/data-tiers/index.md

```diff
@@ -1,10 +1,9 @@
 ---
-id: data-tiers
+slug: /manage/partitions/data-tiers
 title: Data Tiers
 description: Data Tiers provide the ability to allocate data to different storage tiers based on frequency of access - Continuous, Frequent, and Infrequent.
 ---
 
```

Comment:
The content of this article should be placed into a new "Overview" article, and this index article should be updated to contain only a card list of the sub-articles for this section.

Reply:
Currently some sections from this doc is moved to different doc. And we will continue to keep the same doc as a landing page for Data tiers for now, because we have the similar set of page for partitions and flex.